src/app/components/options-sidebar/time-clock/time-clock.component.ts

```diff
@@ -29,6 +29,9 @@ export class TimeClockComponent  implements OnInit {
   seconds: number;
   interval;
   dataTechnology: string;
```

Comment:
technologies entered are going to be a string []

Reply:
done